packages/now-cli/src/util/output/create-output.ts

```diff
@@ -76,6 +77,20 @@ export default function createOutput({ debug: debugEnabled = false } = {}) {
     }
   }
 
+  function spinner(message: string, timeout: number = 300) {
+    if (debugEnabled) {
+      debug(`Spinner invoked (${message})`);
```

Comment:
```suggestion
      debug(`Spinner started (${message})`);
```

Reply:
The spinner doesn't necessarily starts at that point in time. That's why I chose a more generic term here.

Reply:
```suggestion
      debug(`Spinner invoked (${message}) with a ${timeout}ms timeout`);
```